_studio/shared/include/libmfx_core_vaapi.h

```diff
@@ -177,6 +177,8 @@ class VAAPIVideoCORE : public CommonCORE
     // this function should not be virtual
     mfxStatus SetCmCopyStatus(bool enable);
 
+    bool GetCmCopyStatus() const { return m_bCmCopy; }
```

Comment:
Status in function name is misleading. Can we just have `SetCmCopy(bool)` and `bool CmCopy()`?

Reply:
Done.